EmailHandler.php

```diff
@@ -95,22 +95,30 @@ public function validate(array $payload)
             return false;
         }
 
-        // The user must exists in Kanboard
-        $user = $this->userModel->getByEmail($payload['sender']);
-
-        if (empty($user)) {
-            $this->logger->info('Mailgun: ignored => user not found');
-            return false;
-        }
-
         // The project must have a short name
+        //    DMM: Moved project check in front of user check - to support project unknown user mappings
```

Comment:
Not sure if this comment is really useful.

Reply:
Sorry - this was just my note taking - I have removed it